.github/workflows/build.yml

```diff
@@ -21,6 +21,7 @@ jobs:
           - "12-stretch-slim"
           - "14-stretch-slim"
           - "12-buster-slim"
+          - "14-buster-slim"
```

Comment:
It looks like the Dockerfile wasn't added

Reply:
My bad, pushed now